src/models/common/transaction.ts

```diff
@@ -0,0 +1,38 @@
+import { Amount } from ".";
+
+interface CreatedNode {
+    CreatedNode: {
+        LedgerEntryType: string
+        LedgerIndex: string
+        NewFields: {[field: string]: any}
+    }
+}
+
+interface ModifiedNode {
+    ModifiedNode: {
+        LedgerEntryType: string
+        LedgerIndex: string
+        FinalFields: {[field: string]: any}
```

Comment:
Is there any way we can narrow this from `{[field: string]: any`? Does this come in the form of a `LedgerEntry` that we can use to replace `any` here?

Reply:
You could in theory narrow it, but as far as I understand, it's a subset of `LedgerEntry` fields, and isn't necessarily a full `LedgerEntry` object. I suppose this is another question for @intelliot

Reply:
This is probably any (modifiable) fields in any ledger object, so I believe there are a lot of possibilities, and it'll expand in the future as more kinds of ledger objects are added in the future. I would fine with leaving this as-is, but of course, if we want to make this more specific in the future, that would be fine as well.